config/mocha.json

```diff
@@ -13,6 +13,9 @@
   "claim-store": {
     "url": "http://localhost:14400"
   },
+  "feature-toggles-api": {
+    "url": "http://localhost:8580"
```

Comment:
Please make it `18580`. We use different port number in tests to avoid situations that tests work locally but not on Jenkins due to the fact tests hit local dev instance.